docs/api-docs/store-management/shipping/use-real-time-carrier.md

```diff
@@ -0,0 +1,375 @@
+# Use a Real-Time Carrier
+
+Real-time carriers allow merchants and shoppers to obtain **real-time** shipping quotes. There are two types of real-time carriers: native integrations and shipping provider apps. Shipping provider apps require merchants to install the app on their store. Once a merchant installs a shipping provider app on their store, API users and merchants can connect the carrier to the store. API users and merchants can then define and enable a shipping method for a carrier in one or more shipping zones. They can define and enable a different shipping method in each shipping zone for the same carrier. Enabling a zone's real-time shipping method makes the method available to shoppers in that zone during checkout. 
+
+This article is a guide for API users on how to manage carrier connections and real-time shipping methods from carriers using the [Shipping V2 API](/api-reference/store-management/shipping-api).
```

Comment:
what do you intend `API users` to mean?

Reply:
it's people who use the API, so i changed the wording to reflect this